node/as/http.js

```diff
@@ -361,12 +369,11 @@ AsHTTPHandler.prototype.handleRequest = function handleRequest(req, buildRespons
 
     function sendError(err) {
         if (!sent) {
-            // TODO: map type?
             sent = true;
             self.logger.warn('Handling request failed', {
                 error: err
             });
-            buildResponse().sendError('UnexpectedError', err.message);
+            buildResponse().sendError(errors.classify(err), err.message);
```

Comment:
This returns `null` if it cant be classified.

Please default to `'UnexpectedError'` to avoid problems.